uvtools/dspec.py

```diff
@@ -1235,9 +1243,79 @@ def delay_filter_leastsq(data, flags, sigma, nmax, add_noise=False, freq_units =
     return mdl_array, cn_array, inp_data
 
 
-def sinc_downweight_mat_inv(nchan, df, filter_centers, filter_widths,
-                            filter_factors, cache = {}, wrap = False, wrap_interval=1,
-                            nwraps = 1000, no_regularization = False):
+def delay_interpolation_matrix(nchan, ndelay, wgts, fundamental_period=None, cache={}, taper='none', return_diagnostics=False):
+    """
+    Copute a foreground interpolation matrix.
```

Comment:
Small typo (`Copute` -> `Compute`)

Reply:
thanks for catching this :). I've corrected it.